crypto/tss/ecdsa/cggmp/refresh/round_3.go

```diff
@@ -163,12 +163,12 @@ func (p *round3Handler) HandleMessage(logger log.Logger, message types.Message)
 	if !Ai.Equal(Aihat) {
 		return ErrDifferentPoint
 	}
-	// Because a2 = 0, so R can be arbitrary point.
-	err = round3Msg.YschnorrProof.Verify(p.pubKey, ssidSumRho)
+
```

Comment:
Add one variable:
G := pt.NewBase(curve)
err = round3Msg.YschnorrProof.Verify(pt.NewBase(p.pubKey.GetCurve()), ssidSumRho) ->
err = round3Msg.YschnorrProof.Verify(G, ssidSumRho)
and
err = round3Msg.ShareschnorrProof.Verify(pt.NewBase(p.pubKey.GetCurve()), ssidSumRho) ->
err = round3Msg.ShareschnorrProof.Verify(G, ssidSumRho)